data-prepper-plugins/ocsf/src/main/resources/org/opensearch/dataprepper/transforms/templates/ocsf-v1.1-panw-traffic-template.yaml

```diff
@@ -0,0 +1,673 @@
+"<<pipeline-name>>":
```

Comment:
Does the current transformation code support sub-directories? If so, we should probably look at a structure that we can expand over time.

Also, I think we should consider dropping the `rules/` and `transforms/` directories in favor of either a suffix or some other data.

```
transforms/ocsf-v1.1/panw-traffic-rule.yaml
transforms/ocsf-v1.1/panw-traffic-template.yaml
```

If you are working on a transformation, you should be able to modify two files that sit next to each other. Having to work with files in completely different directory structures will become cumbersome as the number of transformations grows.

We can address these items later.

Reply:
Will check with Srikanth and see if sub-directories are supported.